qemu/tests/image_create_with_preallocation.py

```diff
@@ -0,0 +1,78 @@
+import json
+import os
+
+from avocado import fail_on
+from avocado.utils import process
+
+from provider import qemu_img_utils as img_utils
+from virttest import data_dir
+from virttest import qemu_storage
```

Comment:
it might be better:

from virttest import data_dir
from virttest import qemu_storage
 >>> add a new line
from provider import qemu_img_utils as img_utils  >>> under virttest import 

Just keep consistent style

Reply:
The previous rule was in alphabetical order， so put it in front.  I will follow your suggestion, thanks.

Reply:
I see, it might be better to put imports into groups first, and inside each group we can put them in alphabetical order, please refer to https://www.python.org/dev/peps/pep-0008/#imports (But I didn't find the 'alphabetical order rule here)

" Imports should be in sections in the following order: standard library modules, third-party modules, your own modules. Each subsection should have imports in alphabetical order." --- From yongxue's "clean code" ppt

just FYI.